public/pages/CreateMonitor/containers/CreateMonitor/utils/constants.js

```diff
@@ -33,6 +34,22 @@ export const FORMIK_INITIAL_VALUES = {
 
   /* DEFINE MONITOR */
   searchType: 'graph',
+  http: {
+    urlType: URL_TYPE.FULL_URL,
+    url: '',
+    scheme: 'https',
+    host: '',
+    port: '',
+    path: '',
+    queryParams: [
```

Comment:
Should we also provide Headers or it is not needed now ?

Reply:
It is not needed now as I don't have headers on backend.